decidim-elections/app/controllers/concerns/decidim/votings/admin/voting_context.rb

```diff
@@ -1,19 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Well spotted! It seems that other modules as assemblies and conferences are defining this concern even when it is not used :+1:

Reply:
Gotta thank Codecov 😂  Getting rid of it was the only way to make the check pass